src/components/Header.tsx

```diff
@@ -0,0 +1,75 @@
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import DecemberBankIcon from '@mui/icons-material/Paid';
+import { useAppDispatch, useAppSelector } from '../redux/hooks';
+import { getUserInfo } from '../api/account/AccountApi';
+import { loadUserInfo, unLoadUserinfo } from '../redux/slices/accountSlice';
+import { endSession } from '../redux/slices/sessionSlice';
+
+export default function Header() {
+  const navigate = useNavigate();
+  const dispatch = useAppDispatch();
+  const user = useAppSelector((state) => state.account.user);
+  const isAuthenticated = useAppSelector(
+    (state) => state.session.isAuthenticated,
+  );
+
+  const handleHomeClick = (e: any) => {
+    e.preventDefault(e);
```

Comment:
No necesita parámetro
```suggestion
    e.preventDefault();
```